<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_iniciar">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_parar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Parar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ReturnHome">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Return Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CalibSensor">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Calib. sensor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Rotinas</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ganhos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ki</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kd</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kp">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>67</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ki">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>12.5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kd">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Enviar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
